packages/query-core/src/types.ts

```diff
@@ -124,6 +122,26 @@ export interface QueryOptions<
   maxPages?: number
 }
 
+export interface DefaultPageParam<TPageParam = unknown> {
+  defaultPageParam: TPageParam
+}
+
+export interface InfiniteQueryOptions<
```

Comment:
I feel like this name does not convey the purpose of the interface.
Maybe something along `InfiniteQueryPageParamsOptions` would be better?
Since current name could be mistaken for a full set of options, which is not.

Reply:
agree. renamed here: 736edbd9